.github/workflows/tests.yml

```diff
@@ -112,9 +112,14 @@ jobs:
       - name: Run tests for Pydantic v2
         run: |
           python -c "import pydantic; print(pydantic.VERSION)"
-          python -m pytest --pyargs confection -Werror
+          python -m pytest --pyargs confection
 
       - name: Test for import conflicts with hypothesis
         run: |
           python -m pip install hypothesis
           python -c "import confection; import hypothesis"
+
+      - name: Test with spacy
+        run: |
+          python -m pip install spacy
+          python -m spacy init config -p tagger tagger.cfg
```

Comment:
Just a comment that this is only intended to be a temporary addition to the CI. `confection` needs a unit test that catches this issue.